articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+## Configure Azure
+
+::: note
+To complete this, tutorial, you will need to have an account that grants you access to Microsoft's [Azure Portal](https://portal.azure.com).
```

Comment:
I don't get the commas `To complete this, tutorial,`. Probably a typo, you meant to write `To complete this tutorial,`, right?

I'm also thinking that this should probably be a `warning` since having an azure account is very important (can't do without it)